roles/kubernetes/node/defaults/main.yml

```diff
@@ -23,6 +23,10 @@ kubelet_kubelet_cgroups_cgroupfs: "/system.slice/kubelet.service"
 kubelet_fail_swap_on: true
 
 # Reserve this space for kube resources
+# Set to true to reserve resources for kube daemons
```

Comment:
Please add these variables to the sample files as well as they serve a documentation role.

It would also be useful to add some more extensive documentation for these variables in the `docs/` folder.

Reply:
I'm sorry, I misunderstood the yellow "Pending" decorator on my comments, so the above comment was just sent (it was actually commented many days ago).